src/main/java/com/springweb/dv_spring_web_mongo/controller/ProjectController.java

```diff
@@ -14,9 +14,9 @@
     @Autowired
     ProjectService projectService;
 
-    @GetMapping()
-    public List<ProjectDTO> getAllProjects() {
-        return projectService.getAllProjects();
+    @GetMapping
+    public List<ProjectDTO> getAllProjects(@RequestParam(required = false) String param) {
```

Comment:
how I can use the `param`,  please provide example of url for test of this function.

Reply:
I expect that the parameter name will be like this: "filterProjectName"